### PR TITLE
modify version number of deb build in travis

### PR DESCRIPTION
**Pull request by changev**
Modify version number of deb build in travis.

The version number style has been changed according to  buildRelease requirments.
and
Travis doesn't need to build "release" anymore.

This PR is required by https://github.com/RackHD/on-build-config/pull/60
And should be merged before https://github.com/RackHD/on-build-config/pull/60.

@panpan0000 @PengTian0

### CODE REVIEW

File: extra/make-deb.sh [hunk 1]
```diff
@@ -17,30 +17,22 @@ if [ -z "$DEBEMAIL" ]; then
...
-        fi
-        
+        export DCHOPTS="-v ${DEBVERSION} -u low ${DEBPKGVER} -b -m"
 fi
 
 echo "DEBDIR:       $DEBDIR"
 echo "DEBFULLNAME:  $DEBFULLNAME"
 echo "DEBEMAIL:     $DEBEMAIL"
-echo "DEBBRANCH:    $DEBBRANCH"
+echo "DEBVERSION:    $DEBVERSION"
```
Comment (anhou) [inline extra/make-deb.sh]: column is not aligned

File: extra/gen-debversion.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# DEBRANCH format is ${SYMBOL}-${DATESTRING}
+#
+
+DEBDIR="./debian"
+if [ ! -d "${DEBDIR}" ]; then
+    echo "no such debian directory ${DEBDIR}"
+    exit 1
+fi
+
+#Version in changelog will be assigned when branch created
```
Comment (anhou) [inline extra/gen-debversion.sh]: a blank after # is better

File: extra/gen-debversion.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+set -ex
+
+#
+# This script will generate var DEBRANCH
+# DEBRANCH format is ${SYMBOL}-${DATESTRING}
+#
+
+DEBDIR="./debian"
+if [ ! -d "${DEBDIR}" ]; then
+    echo "no such debian directory ${DEBDIR}"
```
Comment (anhou) [inline extra/gen-debversion.sh]: There isn't xxx or  xxx doesn't exist?

**Review by anhou — COMMENTED** [review]
functional logic LGTM, just some minor comments

**Review by yyscamper — APPROVED** [review]

**Comment by panpan0000** [comment]
:+1:

**Comment by panpan0000** [comment]
this solution sounds workable.

**Review by yyscamper — APPROVED** [review]

**Pull request merged**